### ISSUE TITLE
Assets REST API 

### ISSUE THREAD

**Issue by rmoreiraZoi** [atlassian-api/atlassian-python-api#1383]
Hi

I am trying to create assets in JIRA does this library provide any wrapper around the Assets REST API ?

**Comment by Bertsjuhn** [comment]
Yes, it was called Insights before the rename to Assets.
Check the examples at: https://github.com/atlassian-api/atlassian-python-api/tree/master/examples/insight

**Comment by ksbckr** [comment]
Yeah but the URL paths have changed. They have been renamed to assets  which means that this is probably broken.

**Comment by rmoreiraZoi** [comment]
From my testing it seems that it's indeed broken, the correct URL path needs to be rest/assets/ instead of rest/insight/ I would say

**Comment by gonchik** [comment]
added there https://github.com/atlassian-api/atlassian-python-api/commit/8bfa5220ff39df513bb93a4019c4c0daf9ad5ab2
